### PR TITLE
For #1192: Updates detection for Lockbox to Lockwise

### PR DESCRIPTION
**Pull request by sblatz**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/fenix/blob/master/CHANGELOG.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/components/metrics/MozillaProductDetector.kt [hunk 1]
```diff
@@ -22,7 +22,7 @@ object MozillaProductDetector {
         FOCUS("org.mozilla.focus"),
 
         // Other products
-        LOCKBOX("org.mozilla.lockbox")
+        LOCKWISE("org.mozilla.lockwise")
```
Comment (pocmo) [inline app/src/main/java/org/mozilla/fenix/components/metrics/MozillaProductDetector.kt]: The application id hasn't changed!
https://play.google.com/store/apps/details?id=mozilla.lockbox

Comment (sblatz) [reply app/src/main/java/org/mozilla/fenix/components/metrics/MozillaProductDetector.kt]: Thanks for the heads up. Weird that it's failing to find it installed, then 😕

Comment (pocmo) [reply app/src/main/java/org/mozilla/fenix/components/metrics/MozillaProductDetector.kt]: Note that it's `mozilla.lockbox` without `org` :)

Comment (sblatz) [reply app/src/main/java/org/mozilla/fenix/components/metrics/MozillaProductDetector.kt]: Yes, just saw that. Thanks @pocmo

**Pull request closed without merging**